generators/entity-server/templates/src/main/java/package/repository/EntityRepository.java.ejs

```diff
@@ -69,6 +70,7 @@ import java.util.UUID;
 @SuppressWarnings("unused")
 <%_ } _%>
 @Repository
+@Primary
```

Comment:
Not needed if I'm not wrong

Reply:
When there is  another repository inheriting from this one ( say,  I need additional query methods etc ) ,   generated code can not be injected properly.  Alternative solution would be  explicit qualifier.

Reply:
I think that end user would prefer to have their custom repositories picked up instead of the generated ones.
Also, as an alternative regarding custo,  take a look at what has been done on the reactive jdbc repositories

Reply:
Sure,   I also like it this way.  But in case user derived from generated repository, there will be ambiguity  and  generated service can not be autowired properly -  this is a way to resolve  this ambiguity.

Reply:
After some thinking about it - you are correct.  If user derives from the generated repository he can place @Primary on  hos own class and resolve ambiguity this way.